components/sidebar/channel_navigator/channel_navigator.tsx

```diff
@@ -53,47 +60,79 @@ export default class ChannelNavigator extends React.PureComponent<Props, State>
             channelSwitchTextShortcutDefault = '⌘K';
         }
 
-        let historyArrows;
-        if (isDesktopApp()) {
-            historyArrows = (
-                <React.Fragment>
-                    <button
-                        className={classNames('SidebarChannelNavigator_backButton', {disabled: !this.props.canGoBack})}
-                        disabled={!this.props.canGoBack}
-                        onClick={this.goBack}
-                        aria-label={Utils.localizeMessage('sidebar_left.channel_navigator.goBackLabel', 'Back')}
-                    >
-                        <i className='icon icon-arrow-left'/>
-                    </button>
-                    <button
-                        className={classNames('SidebarChannelNavigator_forwardButton', {disabled: !this.props.canGoForward})}
-                        disabled={!this.props.canGoForward}
-                        onClick={this.goForward}
-                        aria-label={Utils.localizeMessage('sidebar_left.channel_navigator.goForwardLabel', 'Forward')}
-                    >
-                        <i className='icon icon-arrow-right'/>
-                    </button>
-                </React.Fragment>
-            );
-        }
+        const jumpToButton = (
+            <button
+                className={'SidebarChannelNavigator_jumpToButton'}
+                onClick={this.openQuickSwitcher}
+                aria-label={Utils.localizeMessage('sidebar_left.channel_navigator.channelSwitcherLabel', 'Channel Switcher')}
+            >
+                <i className='icon icon-magnify'/>
+                <FormattedMessage
+                    id='sidebar_left.channel_navigator.jumpTo'
+                    defaultMessage='Find channel'
+                />
+                <div className={'SidebarChannelNavigator_shortcutText'}>
+                    {channelSwitchTextShortcutDefault}
+                </div>
+            </button>
+        );
 
-        return (
-            <div className={'SidebarChannelNavigator'}>
+        const addChannelDropdown = (
+            <AddChannelDropdown
+                showNewChannelModal={this.props.showNewChannelModal}
+                showMoreChannelsModal={this.props.showMoreChannelsModal}
+                showCreateCategoryModal={this.props.showCreateCategoryModal}
+                canCreateChannel={this.props.canCreateChannel}
+                canJoinPublicChannel={this.props.canJoinPublicChannel}
+            />
+        );
+
+        const historyArrows = (
```

Comment:
Minor nit, can you move this into the desktop app branch of the if statement below?